demomultistoreform/README.md

```diff
@@ -10,6 +12,33 @@ This module demonstrates how to make your forms multistore compatible in your mo
 - A fixtures generation mechanism, using [doctrine](https://devdocs.prestashop.com/1.7/modules/concepts/doctrine/)
 - Displaying the configured blocks on Front Office
 
+### Multistore features
+
+#### The shop association block 
+
+You should display it as soon as the user wishes to add an element. It allows him to choose with which store(s) he wants to add the element.
+
+#### Listing
+
+The elements displayed in the listing depend on the store selected in the context.
+The all shops context displays the elements of all the shops. 
+
+#### Checkboxes
+
+The user should be able to apply a specific configuration for a specific shop or group. To do this, checkboxes should be added before each element when a specific shop context or a group is selected as context.
+
+#### Specific settings drop-down
+
+The user should be able to see which parameters have a specific configuration for a shop or a group when he is in all shops context. This information let him know that the change he will make in all shops context will not be applied for shops that already have a parameter configured specifically. To do this, specific settings drop-down should be added in all shops context and in group context.
+ 
+#### Enable / disable from the listing
+
+The enable / disable action made from the listing enable/disable the element for all the stores with which it’s associated and not only for the context selected.
+
+#### Delete from the listing
+
+If an element is associated with several shops, then if I delete it from the listing, it will be disassociated from the selected context(s) and deleted if it is no longer associated with any store.
```

Comment:
```suggestion
If an element is associated with several shops, then a user deletes it from the listing, it will be dissociated from the selected context(s) and deleted if it is no longer associated with any store.
```